WebContent/js/components/Snackbar.js

```diff
@@ -49,7 +49,11 @@ class AtlasSnackbar extends React.Component {
         }, this.notificationDuration);
     }
 
-    handleRequestClose = () => {
+    handleRequestClose = (event, reason) => {
+        if (reason === 'clickaway') {
+            return;
```

Comment:
If we return here without the logic to pop the message and clear the timeout won't this leave the snackbar in an unknown state?

Reply:
this is just to stop closing the pop on click outside the popup, which can make people miss the notification. Disabling that scenario just for click away reason, for all other cases it will complete the handler, and clear timeout.
- on timeout
- on close on notification popup

Reply:
there is some snackbar notification behavior investigating though

Reply:
popmessage() -> not being called on clickaway did cause some out of sync between redux
removed logic for now.

Though updated componentWillReceiveProps to componentDidUpdate as recommended for react 16+